README.md

```diff
@@ -271,7 +195,8 @@ Are you using GitLink in your projects? Let us know and we will add your project
 
 *Note that you can also create a pull request on this document and add it yourself.* 
  
-
 # Icon
 
 Link by Dominic Whittle from The Noun Project
+
+[NuGetDownload]: https://www.nuget.org/packages/gitlink
```

Comment:
This is the definition of the link. At the top, we consume it. Are you familiar with the markdown link syntax where URLs are defined at the bottom of the document?